ansible/server-state-playbooks/nightshade-web/playbook.yml

```diff
@@ -42,18 +23,35 @@
       lvm_lvfilesystem: "{{ provision_root_filesystem }}"
 
     # OMERO.web configuration in host_vars in different repository
-    # Upgrade 5.2.8 Web to 5.3.0
     - role: openmicroscopy.omero-web
       omero_web_release: 5.3.2
-      omero_web_upgrade: True
       no_log: true
 
+    # This to be refactored for RHEL/non-spacewalk at some point
```

Comment:
If you refactor I can think of two options:
- Install/configure `openmicroscopy.system-monitor-agent` in each server's playbook as at present
- Remove all mentions of `openmicroscopy.system-monitor-agent` in individual playbooks and have a single site-wide playbook that handles internal monitoring on all servers.